### ISSUE TITLE
Revisit Integration List Page UX design 

### ISSUE THREAD

**Issue by dongniwang** [syndesisio/syndesis#3892]
## This is a...

<pre><code>
[ x ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Documentation issue or request
</code></pre>

## The problem

The Configuration Required label wraps into a new line right beneath the integration publishing progress bar with no breathing room in between. 

Configuration Required label is occupying a column without providing valuable informaiton at a glance.  

## Expected behavior

Invesitgate design options to address the above problems. 

## Screenshot

<img width="1218" alt="screen shot 2018-10-17 at 2 56 58 pm" src="https://user-images.githubusercontent.com/24943812/47109692-eeac8100-d21c-11e8-87bf-5e1a3cac0f7e.png">

**Comment by gashcrumb** [comment]
Maybe we don't need to show the description, which would provide a lot more space.

**Comment by gashcrumb** [comment]
On other thing is we could also just hide the "configuration required" bit while a deployment is happening, as TBH the user can't do much about it until the deployment is finished.  Except stop or delete the integration I guess.

**Comment by gashcrumb** [comment]
True!   My only other follow on thoughts:

* it's at least located right next to the state/status which I consider to be a primary thing to look at on this page.  
* it's got a warning icon which I think is fairly eye catching.

Maybe it's just too wordy?
